gladys-assistant/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  web:
+    image: gladysassistant/gladys:v4
+    restart: on-failure
+    stop_grace_period: 1m
+    privileged: true
+    network_mode: host
+    cgroup: host
+    volumes:
+      - ${APP_DATA_DIR}/gladysassistant:/var/lib/gladysassistant
```

Comment:
```suggestion
      - ${APP_DATA_DIR}/data/gladysassistant:/var/lib/gladysassistant
```

This is where app data is usually located

Reply:
`${APP_DATA_DIR}` points to the `gladys-assistant` parent directory shown here:
<img width="303" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/1445094b-1956-4898-ac70-491f9e99cbc9">

The `data` directory under `gladysassistant` is at `${APP_DATA_DIR}/data/` as @highghlow shows. 

If we want to bind `/var/lib/gladysassistant` to the host filesystem at `${APP_DATA_DIR}/data/gladysassistant` (which I think is a good idea), then you will need to add a `gladysassistant` directory under the data directory and add a `.gitkeep` file to it.

This will future proof the data directory in the event that you add other containers to this app in the future, or you want to bind other directories in the future.

Reply:
Let me check and fix that

Reply:
I can do this and push a commit @cicoub13

Reply:
multi-arch image digest added here: https://github.com/getumbrel/umbrel-apps/pull/1044/commits/c20b58509b008748452d7aca6b30e2047a14bc8b

An easy way to grab the digest when you know your image is multi-architecture is to run 
```
docker pull <image>:<tag>
```

and you will see the multi-arch digest in the output. For example:
```
$ sudo docker pull gladysassistant/gladys:v4
v4: Pulling from gladysassistant/gladys
...
Digest: sha256:f694d49bf57426cfc37590eb71a19b0e183cd3e8cbae966ef093bb3d199d318b
Status: Downloaded newer image for gladysassistant/gladys:v4
docker.io/gladysassistant/gladys:v4
```